### PR TITLE
Connectors v2 Feature

### PR DESCRIPTION
**Pull request by wtrocki**
Feature branch for connectors v2 commands

### CODE REVIEW

**Comment by wtrocki** [comment]
#requires rebase

**Comment by jackdelahunt** [comment]
@wtrocki are we merging this?

**Comment by wtrocki** [comment]
@jackdelahunt that is feature branch. Feature branch are rebased periodically. 
They can be used to review strings and aggregate all PRs. This would be merged when we finish **all** requirements for phase2. We have PR for visibility and collaboration.

File: pkg/core/localize/locales/en/cmd/connectors.toml [hunk 1]
```diff
@@ -294,4 +294,57 @@ rhoas connector namespace delete --id jdhdhdhmmf
...
+Select a connector instance to be the current instance. When you set the connector instance to be used, it is set as the current instance for all 
+“rhoas connector cluster” commands.
+
+You can select a  connector instance by name or ID.
+'''
+
+[connector.use.cmd.example]
+description = 'Examples of how to use the command'
+one = '''
+# Select a connector instance by name to be set in the current context
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/connectors.toml:373]: Set the current Connectors instance by providing the name of a Connectors instance.

File: pkg/core/localize/locales/en/cmd/connectors.toml [hunk 2]
```diff
@@ -294,4 +294,57 @@ rhoas connector namespace delete --id jdhdhdhmmf
...
+$ rhoas connector use --name=my-connector
+
+# Select a connector instance by ID to be set in the current context
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/connectors.toml:376]: Set the current Connectors instance by providing the ID of a Connectors instance.

File: pkg/core/localize/locales/en/cmd/connectors.toml [hunk 3]
```diff
@@ -294,4 +294,57 @@ rhoas connector namespace delete --id jdhdhdhmmf
...
+$ rhoas connector use --id=1iSY6RQ3JKI8Q0OTmjQFd3ocFRg
+'''
+
+[connector.common.error.idNotFound]
+description = "Error message for when and connector with ID is not found"
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/connectors.toml:381]: Error message for when a Connectors instance ID is not found

File: pkg/core/localize/locales/en/cmd/connectors.toml [hunk 4]
```diff
@@ -294,4 +294,57 @@ rhoas connector namespace delete --id jdhdhdhmmf
...
+one = "Connector instance with ID {{.ID}} not found"
+
+[connector.common.error.nameNotFound]
+description = "Error message for when and connector with name is not found"
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/connectors.toml:385]: Error message for when a Connectors instance name is not found

File: pkg/core/localize/locales/en/cmd/context.en.toml [hunk 5]
```diff
@@ -121,6 +121,16 @@ $ rhoas context set-kafka --name=my-kafka
 $ rhoas context set-kafka --id=1iSY6RQ3JKI8Q0OTmjQFd3ocFRg
 '''
 
+[context.setConnector.cmd.example]
+description = 'Examples of how to use the command'
+one = '''
+# Select a connector instance by name to be set in the current context
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/context.en.toml:127]: Set the current Connectors instance by providing the name of a Connectors instance.

File: pkg/core/localize/locales/en/cmd/context.en.toml [hunk 6]
```diff
@@ -121,6 +121,16 @@ $ rhoas context set-kafka --name=my-kafka
...
+$ rhoas context set-connector --name=my-connector
+
+# Select a connector instance by ID to be set in the current context
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/context.en.toml:130]: Set the current Connectors instance by providing the ID of a Connectors instance.

File: pkg/core/localize/locales/en/cmd/context.en.toml [hunk 7]
```diff
@@ -233,6 +243,12 @@ The context doesn't have a Kafka instance ID.
 Use the "rhoas context set-kafka" command to select a Kafka instance for your context
 '''
 
+[context.common.error.noConnectorID]
+one='''
+The rhoas context doesn't have a connector instance ID.
+Use the "rhoas context set-connector" command to select a connector instance for your context
```
Comment (MelissaFlinn) [inline pkg/core/localize/locales/en/cmd/context.en.toml:249]: The Connectors instance for the current context is not set. The `rhoas` CLI uses the current Connectors instance when you run any `rhoas connector cluster` commands.
Use the `rhoas context set-connector` command to set the current Connectors instance.

**Review by MelissaFlinn — CHANGES_REQUESTED** [review]
Looks good overall. Let me know if you have any questions...

**Comment by MelissaFlinn** [comment]
Is there/will there be a command that the user can run that returns the name and ID of the current Connectors instance?

**Comment by jackdelahunt** [comment]
@MelissaFlinn We added connectors to the cli's context so the current working connector is always known and the user can get details about it easily with `rhoas context status connector`. They can also get details of all connectors they have with `rhoas connector list`.

**Pull request merged**